packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -349,24 +350,19 @@ export class SummarizerNode implements IRootSummarizerNode {
 							latestSummaryUpdated: true,
 							wasSummaryTracked: true,
 							summaryRefSeq,
+							refreshScenario:
+								"Successfully refreshed summary ack for tracked summary",
 						};
 					}
-
-					const props = {
-						summaryRefSeq,
-						pendingSize: this.pendingSummaries.size ?? undefined,
-					};
-					this.logger.sendTelemetryEvent({
-						eventName: "PendingSummaryNotFound",
-						proposalHandle,
-						referenceSequenceNumber: this.referenceSequenceNumber,
-						details: JSON.stringify(props),
-					});
 				}
 
 				// If the summary for which refresh is called is older than the latest tracked summary, ignore it.
 				if (this.referenceSequenceNumber >= summaryRefSeq) {
 					eventProps.latestSummaryUpdated = false;
+					eventProps.refreshScenario =
+						this.referenceSequenceNumber === summaryRefSeq
+							? "Received ack matching the current referenceSequenceNumber without a tracked summary"
```

Comment:
Do we need these long text messages ? Would it be possible to decrease the size of these text messages?  Doesn't it increase the bundle size ? + COGS ?

Reply:
Yeah, I'm not sure we want to do this. This just explains what happens. Maybe we can mark these with numbers and then put comments in the code?

Reply:
Looks like there are 7 different scenarios, each slightly different.

Reply:
Another thought is that since we are moving away from using refresh, we might not need any of these telemetry changes. I'm going to put this in Draft Form.